sqlitecluster/SQLiteSequentialNotifier.h

```diff
@@ -53,7 +53,7 @@ class SQLiteSequentialNotifier {
     uint64_t _value;
 
     // If there is a global result for all pending operations (i.e., they've been canceled), that is stored here.
-    RESULT _globalResult;
+    atomic<RESULT> _globalResult;
```

Comment:
If you change this and the issue appears to go away, we won't know if it's actually fixed or not. I'd love to do blue/green with just the log line and then add this after, but I'm not sure how easy it is to reproduce. What do you think?